img_src/Oxygen_Icon_Set/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+Oxygen Icon Theme SVG Sources
```

Comment:
Please uncapitalize the name of this directory, i.e. leave it as `oxygen_icon_set`.

Reply:
Okay, sure.

FIXME

Reply:
Done